### PR TITLE
[MWPW-132804] Fix CC checks & cleanup NALA small issues

### PR DESCRIPTION
**Pull request by chadsunvice**
This PR refers to ticket [MWPW-132804](https://jira.corp.adobe.com/browse/MWPW-132804) Fix CC checks & cleanup NALA small issues.

### CODE REVIEW

**Review by JackySun9 — APPROVED** [review]

File: tests/cc/febpub.test.js [hunk 1]
```diff
@@ -1,13 +1,38 @@
+/* eslint-disable import/named */
+/* eslint-disable no-await-in-loop */
+/* eslint-disable import/extensions */
 /* eslint-disable no-restricted-syntax */
```
Comment (skumar09) [inline tests/cc/febpub.test.js:4]: - I would not suggest disabling eslint rule:` /* eslint-disable no-restricted-syntax */ `at the global level.

**Review by skumar09 — APPROVED** [review]
- All looks good, except disabling eslint rule : `/* eslint-disable no-restricted-syntax */`
- Please update the above before merging.

File: tests/cc/febpub.test.js (hunk 1 above)
Comment (chadsunvice) [reply tests/cc/febpub.test.js]: Done!

**Review by skumar09 — APPROVED** [review]

**Review by amauch-adobe — APPROVED** [review]
Approved.

**Pull request merged**